doc/GETSTARTED.md

```diff
@@ -1,88 +1,102 @@
 # Get Started
 
-- Skill requirements
+## Skill requirements
```

Comment:
Is here any change except of reformatting? In which commits?

Reply:
Since every other markdown was using headings, we replaced the list syntax with heading syntax and introduced some links at the end of the file, to help people dig down into various other tutorials instead of navigating through the OS directory or github directory structure.